resources/app.py

```diff
@@ -633,6 +1055,10 @@ def create_operating_room_reservation(
     operating_room_reservation: create_operating_room_reservation_schema,
     db=Depends(get_db),
 ):
+    operating_room_reservation = operating_room_reservation_model(
```

Comment:
One thing i have noticed (probably i created this bug):

```
[
  {
    "ID_operating_room": 1,
    "Reservation_date": "2024-04-22",
    "End_time": "13:30:00",
    "Start_time": "11:30:00",
    "ID_reservation": 1,
    "ID_plan": 1
  },
  {
    "ID_operating_room": 1,
    "Reservation_date": "2024-04-22",
    "End_time": "14:30:00",
    "Start_time": "10:30:00",
    "ID_reservation": 2,
    "ID_plan": 1
  }
]
```

I succedeed in creating reservation that is inside another reservation (we can check if same bug is in bed reservation)